docs/docs-content/release-notes.md

```diff
@@ -78,6 +83,25 @@ Palette 4.2.0 is a release that includes new features and various improvements.
 - Extended [kube-vip customization](https://kube-vip.io/docs/installation/flags/) is now available for new Edge clusters. You can now specify additional kube-vip configuration parameters as part of the Kubernetes pack layer configuration. To learn more about the available kube-vip configuration parameters, refer to the [Publish Cluster Services with Kube-vip](clusters/edge/networking/kubevip.md) resource.
 
 
+
+#### Known Issues
+
+- The following known issues apply to the VxLAN network overlay feature:
+
+  - When adding multiple nodes to an existing cluster with overlay enabled, failure to add one node will block the addition of the other nodes.
+
+  - When deleting an Edge host from a cluster with overlay enabled, ensure the node doesn't have the `palette-webhook` pod on it, or the node will be stuck in the deleting state. You can use the command `kubectl get pods --all-namespaces --output wide` to identify which node the pod `palette-webhook` is on. Reach out to our support team [support@spectrocloud.com](mailto:support@spectrocloud.com) if you need to remove a node with the `palette-webhook` pod on it.
+
+- In a multi-node cluster with [PXK-E](./integrations/kubernetes-edge.md) as the Kubernetes distribution, you cannot change the Network Interface Card (NIC). When you add an Edge host to such a cluster, leave the NIC field as its default value.
+
+- The following known issues apply to [Harbor Edge Native Config](./integrations/harbor-edge.md) when deployed with the [Longhorn](./integrations/longhorn.md) Container Storage Interface (CSI) driver:
+
+  - The Harbor job service pod is in a *Terminating* and *ContainerCreating* state in an Edge Native High Availability (HA) cluster after a Day-2 operation.
+
+  - The Harbor core, database, and jobs service pods are in a state of CrashLoopBackOff on a long-running cluster with a single node using RKE2.
```

Comment:
My understanding is that core and job service weren't starting because DB wouldn't start. The engineers discussed this in [this ticket](https://spectrocloud.atlassian.net/browse/PE-3337) and I summarized their findings to be a little more specific. And even though this issue was discovered with the cluster using RKE2 single node, it could happen in other environments as well. 
```suggestion
  - The Harbor database pod might fail to start due to file permission issues. This is a [known issue](https://github.com/goharbor/harbor-helm/issues/1676) in the Harbor GitHub repository.
```